backend/pkg/helm/release.go

```diff
@@ -619,8 +641,13 @@ func (h *Handler) UpgradeRelease(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = h.setStatus("upgrade", req.Name, processing, nil)
+	if err != nil {
+		zlog.Error().Err(err).Str("releaseName", req.Name).Str("action", "upgrade").Msg("failed to set status")
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+
```

Comment:
```suggestion
if err = h.setStatus("upgrade", req.Name, processing, nil); err != nil {
		zlog.Error().Err(err).Str("releaseName", req.Name).Str("action", "upgrade").Msg("setting status")
		http.Error(w, err.Error(), http.StatusInternalServerError)
	}
```

Reply:
updated the error message but didn't change the if block style, as this style is used across.